scripts/run_e2e_tests.sh

```diff
@@ -107,9 +108,11 @@ maestro "$target" \
   --env MAESTRO_MOCK_ANALYTICS_SERVER="http://localhost:$MOCK_ANALYTICS_SERVER_PORT" \
   --env MAESTRO_NUMBER_PHONE="0607080910" \
   --env MAESTRO_PASSWORD="$password" \
+  --env MAESTRO_RUN_TRACKING_TESTS="$tracking_tests_value" \
```

Comment:
toutes les variables sont des valeurs

j'aimerais éviter qu'on suffixe toutes variables de `_value`

je trouve ça bizarre d'avoir `_value` dans le nom d'une variable, je propose

```suggestion
  --env MAESTRO_RUN_TRACKING_TESTS="$run_tracking_tests" \
```

Reply:
Pas bien du tout oui, je modifie ça pour le commit de review ✅